BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/security/rest/GroupController.java

```diff
@@ -22,4 +26,10 @@ public UserGroupDto createGroup(@RequestBody UserGroupDto userGroupDto) {
         UserGroup group = groupService.createGroup(groupsMapper.toGroup(userGroupDto));
         return groupsMapper.toDto(group);
     }
+
+    @GetMapping
+    @PreAuthorize("isAuthenticated()")
+    public List<UserGroupDto> getGroups(@AuthenticationPrincipal uk.ac.ebi.biostd.authz.User user) {
```

Comment:
I don't see any other `User` class being imported. Why do you use the full class path?